bin/build-electron

```diff
@@ -104,22 +122,22 @@ export MACOSX_DEPLOYMENT_TARGET="10.10"
 
     if [ "${BUILD_TYPE}" = "debug" ]
     then
-        npm_config_arch=x64 npm_config_target_arch=x64 npm_config_disturl=https://atom.io/download/electron npm_config_runtime=electron npm_config_target=11.2.0 npm_config_build_from_source=true npm_config_devdir=~/.electron-gyp cargo build --features electron
+        npm_config_arch=${TARGET_ARCH} npm_config_target_arch=${TARGET_ARCH} npm_config_disturl=https://atom.io/download/electron npm_config_runtime=electron npm_config_target=11.2.0 npm_config_build_from_source=true npm_config_devdir=~/.electron-gyp RUSTFLAGS="-C target-feature=+crt-static" cargo build --target ${CARGO_TARGET} --features electron
     else
-        npm_config_arch=x64 npm_config_target_arch=x64 npm_config_disturl=https://atom.io/download/electron npm_config_runtime=electron npm_config_target=11.2.0 npm_config_build_from_source=true npm_config_devdir=~/.electron-gyp RUSTFLAGS="-C link-arg=-s" cargo build --features electron --release
+        npm_config_arch=${TARGET_ARCH} npm_config_target_arch=${TARGET_ARCH} npm_config_disturl=https://atom.io/download/electron npm_config_runtime=electron npm_config_target=11.2.0 npm_config_build_from_source=true npm_config_devdir=~/.electron-gyp RUSTFLAGS="-C link-arg=-s -C target-feature=+crt-static" cargo build --target ${CARGO_TARGET} --features electron --release
```

Comment:
Had to add `target-feature=+crt-static`, because the `ia32` build was throwing linker errors for `msvcrt`. According to a Rust RFC it is supported to add `target-feature=+crt-static`: https://github.com/rust-lang/rfcs/blob/master/text/1721-crt-static.md#customizing-linkage-to-the-c-runtime